<?xml version="1.0"?>
<launch>
  <arg name="config_path" default="$(find config)/vehicle" />
  <arg name="sensor_meta" default="$(arg config_path)/sensor/calibrated_sensor.pb.txt" />
  <arg name= "perception_path" default="$(find rs_perception_node)/config" />
  <node name="rs_perception_node" ns="/perception/lidar"
    pkg="rs_perception_node" type="rs_perception_node" output="screen"
    args="--logtostderr=0 --stderrthreshold=1 --v=5
          --sensor_meta_path=$(arg sensor_meta)
          --lidar_segmentation_conf_path=$(arg config_path)/perception/lidar/lidar_segmentation.pb.txt
          --publish_lidar_cluster=true
          --lidar_input=c32_front_left;c32_front_right" >
	  <param name="perception_config_path" type="string" value="$(arg perception_path)" />
  </node>

  <node name="perception_lidar" ns="perception/lidar"
    pkg="perception_lidar" type="perception_lidar_node" output="screen"
    args="--logtostderr=0 --stderrthreshold=1 --v=5
          --lidar_obstacle_topic=/perception/lidar/lidar_obstacle_cluster
          --sensor_meta_path=$(arg sensor_meta)
          --lidar_segmentation_conf_path=$(arg config_path)/perception/lidar/lidar_segmentation.pb.txt
          --publish_lidar_cluster=true
          --lidar_input=c32_front_left;c32_front_right" />
</launch>

